src/shared/cgroup-setup.c

```diff
@@ -17,6 +17,67 @@
 #include "fileio.h"
 #include "user-util.h"
 #include "fd-util.h"
+#include "virt.h"
+
+static bool cg_is_unified_possible(void) {
+        _cleanup_free_ char *buf = NULL;
+        _cleanup_strv_free_ char **lines = NULL;
+        char **line;
+        int r;
+
+        r = read_full_virtual_file("/proc/cgroups", &buf, NULL);
+        if (r < 0) {
+                log_error_errno(r, "Could not read /proc/cgroups, ignoring.");
```

Comment:
always include the error cause, i.e. suffix the log message with `…, ignoring: %m` 

also, errors we ignore should be at warning level at most, but given that this one is "library-like" i'd dowgrade it to `log_debug_errno()` all the way